test/strategy-test.ts

```diff
@@ -161,18 +161,33 @@ export function doStrategyTest(test_case: TestableStrategy) {
             await SnowGlobe.connect(walletSigner).earn();
 
             await fastForwardAWeek();
-                
-            //let harvestable = await Strategy.getHarvestable();
-            let harvestable = await Strategy.getWavaxAccrued();
-            log(`\tHarvestable, pre harvest: ${harvestable.toString()}`);
-            let initialBalance = await Strategy.balanceOf();
-            await Strategy.connect(walletSigner).harvest();
-            await increaseBlock(2);
-            //harvestable = await Strategy.getHarvestable();
-            harvestable = await Strategy.getWavaxAccrued();
-            log(`\tHarvestable, post harvest: ${harvestable.toString()}`);
-
-            return [amt, initialBalance];
+            
+            let harvestable: string;
+            if (Strategy.functions['getHarvestable']){

```

Comment:
There's no benefit to duplicating code that is not affected by `getHarvestable` | `getWavaxAccrued` execution.

See my previous comment